### DISCUSSION TITLE
Netbeans 13 hanging on save

### DISCUSSION THREAD

**Discussion by MatthewCooper-NHM** [apache/netbeans#3791] [General]
Hello. I am trying out Netbeans 13 for a Maven project I am working on. Unfortunately Netbeans is often freezing when saving a Java file. It is a complete freeze - I have to kill Netbeans via the Windows task manager. I am trying to reliably reproduce the problem so that I can raise a more constructive bug report, but am having difficulty nailing it down. Is there anything anyone could recommend trying? 

I can see in the netbeans log this message after it happens:

`INFO [org.netbeans.modules.java.source.parsing.VanillaPartialReparser]: Incorrect partial reparse detected, dump filed: ...`

Thank you!

**Comment by Chris2011** [comment]
You can share the file with us. You can also try to deactivate some plugins or just start with a fresh installation and try to reproduce it step by step if possible.

**Reply by MatthewCooper-NHM** [reply]
Thank you for your response.

I have turned off everything in the "On Save" options for java files and the problem seems to have gone away. I'll keep trying to narrow it down if I can.

The project I'm working on is quite large so I'll try to find some time to create a small example.

Cheers!
